kairon/shared/cognition/processor.py

```
@@ -1,17 +1,26 @@
+import json
 from datetime import datetime
+from pathlib import Path
 from typing import Text, Dict, Any, List
 import json
```

Ruff (0.8.2)

Redefinition of unused `json` from line 1

Remove definition: `json`

(F811)